src/content/docs/en/anywhere/xr.mdx

```diff
@@ -10,4 +10,4 @@ import Alert from "../../../../components/Docs/Alert.astro";
   page is subject to change.
 </Alert>
 
-Would use [StardustXR](https://stardustxr.org/). Our packaging effort can be found [here.](https://github.com/Ultramarine-Linux/stardustxr)
+Would use [StardustXR](https://stardustxr.org/). StardustXR is fully packaged in Terra since Fedora 41
```

Comment:
`stardust-server`: https://github.com/terrapkg/packages/pull/2048

PR still open

Reply:
Should be merged by UM41, gonna look at this later

Reply:
> `stardust-server`: https://github.com/terrapkg/packages/pull/2048
> 
> 
> 
> PR still open

Should be merged by UM41, gonna look at this later

Reply:
Also missing `.`